.chloggen/coralogix-batch-export.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: coralogixexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "improve coralogix exporter batching support"
```

Comment:
Reading through the changes, I don't believe the changes are improving batching support (it may do that internally or vendor side) but the user facing changes is that, _Changing Application metadata to use Resource Attributes_, or something to that affect.

With this change, do users now need to add in the application name and system name manually as part of their resource attributes? Or is it expected to be already done.

Reply:
Right, So the user facing change is that improved performance sending data?

Sorry, I would like to help keep the changelog rather user focused since if you were to imagine a user who is rather new in the career may not come across batching yet but improved performance is easy to grasp as a win.

If we keep the text as a short, simple statement as improves performance and the subtext as the technical change then both ends of the user spectrum are clear on the changes.